ibc-clients/ics07-tendermint/src/client_state/common.rs

```diff
@@ -120,9 +134,49 @@ pub fn verify_consensus_state(consensus_state: Any) -> Result<(), ClientError> {
         });
     };
 
+    if consensus_state_status(
+        &tm_consensus_state.timestamp().into(),
+        host_timestamp,
+        trusting_period,
+    )?
+    .is_expired()
+    {
+        return Err(ClientError::ClientNotActive {
+            status: Status::Expired,
+        });
+    }
+
     Ok(())
 }
 
+pub fn consensus_state_status(
+    consensus_timestamp: &Timestamp,
+    host_timestamp: &Timestamp,
+    trusting_period: Duration,
```

Comment:
There's nothing in this function's signature that actually enforces whether the `Status` it returns is actually associated with a consensus state. Not sure that's a valid criticism that warrants any changes to this implementation, but thought I would mention it.

Reply:
Makes sense. See if [2c6b8e8](https://github.com/cosmos/ibc-rs/pull/1294/commits/2c6b8e8e818c2002b6f4cc63099d9660b670fd7f) works?